cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1080,14 +1080,8 @@
     </property>
 
     <property>
-      <name>security.auth.server.address</name>
-      <value></value>
-      <description>Deprecated. Use security.auth.server.bind.address instead.</description>
-    </property>
-
-    <property>
-      <name>security.auth.server.bind.address</name>
-      <value>0.0.0.0</value>
+        <name>security.auth.server.bind.address</name>
+        <value>0.0.0.0</value>
```

Comment:
In the docs, we currently give 127.0.0.1 as the default; which should it be?
See
http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT/en/admin-manual/installation/cdap-site.html